src/backend/joanie/settings.py

```diff
@@ -494,6 +494,11 @@ class Base(Configuration):
     BREVO_COMMERCIAL_NEWSLETTER_LIST_ID = values.IntegerValue(
         None, environ_name="BREVO_COMMERCIAL_NEWSLETTER_LIST_ID", environ_prefix=None
     )
+    BREVO_WEBHOOK_TOKEN = values.Value(
+        None, environ_name="BREVO_WEBHOOK_TOKEN", environ_prefix=None
+    )
+    if BREVO_API_KEY is not None:
+        JOANIE_AUTHORIZED_API_TOKENS.value.append(BREVO_API_KEY)
```

Comment:
I'm pretty sure that is not working.
IMO, you should put this logic into the `post_setup` callback.

Reply:
Indeed, if JOANIE_AUTHORIZED_API_TOKENS is defined through an environment variable this is not working.

```py
In [1]: from django.conf import settings

In [2]: settings.BREVO_WEBHOOK_TOKEN
Out[2]: 'secretTokenBrevo'

In [3]: settings.JOANIE_AUTHORIZED_API_TOKENS
Out[3]: ['secretTokenForRemoteAPIConsumer']

```

```py
In [1]: from django.conf import settings

In [2]: settings.JOANIE_AUTHORIZED_API_TOKENS
Out[2]: ['secretTokenBrevo']
```

Reply:
Nice catch, should be fixed now.